embulk-core/src/test/java/org/embulk/spi/time/TestRubyTimeFormat.java

```diff
@@ -1,372 +0,0 @@
-package org.embulk.spi.time;
```

Comment:
The same tests in https://github.com/embulk/embulk-util-rubytime/blob/v0.3.2/src/test/java/org/embulk/util/rubytime/TestFormat.java